burn-import/pytorch-tests/tests/missing_module_field/mod.rs

```diff
@@ -0,0 +1,30 @@
+use burn::{module::Module, nn::conv::Conv2d, tensor::backend::Backend};
+
+#[derive(Module, Debug)]
+pub struct Net<B: Backend> {
+    do_not_exist_in_pt: Conv2d<B>,
+}
+
+#[cfg(test)]
+mod tests {
+    type Backend = burn_ndarray::NdArray<f32>;
+
+    use burn::record::{FullPrecisionSettings, Recorder};
+    use burn_import::pytorch::PyTorchFileRecorder;
+
+    use super::*;
+
+    #[test]
+    #[should_panic(
+        expected = "Cannot deserialize Conv2dRecordItem struct with default values orinated from 'do_not_exist_in_pt' field"
```

Comment:
If orinated -> originated as in my previous comment, should also modify here accordingly.